services/wallet/router/router_v2.go

```diff
@@ -133,6 +134,10 @@ type PathV2 struct {
 
 type ProcessorError struct {
 	ProcessorName string
+	SendType      SendType
+	FromChainID   uint64
+	ToChain       uint64
```

Comment:
where is all this extra data used?

Reply:
Added that to try some other approach for errors, but didn't go, then it left here somehow, removed now. Thanks.